<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParserPage</class>
 <widget class="QWidget" name="ParserPage">
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form1</string>
  </property>
  <layout class="QVBoxLayout" name="_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Offline Emulation</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="3">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="roomNameColorLabel">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>MS Shell Dlg</family>
          <pointsize>8</pointsize>
          <italic>false</italic>
          <bold>true</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="text">
         <string>roomname</string>
        </property>
        <property name="margin">
         <number>3</number>
        </property>
        <property name="indent">
         <number>3</number>
        </property>
        <property name="buddy">
         <cstring>roomNameColorPushButton</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="roomNameColorPushButton">
        <property name="text">
         <string>Select</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Room name color:</string>
        </property>
        <property name="buddy">
         <cstring>roomNameColorPushButton</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Descriptions color:</string>
        </property>
        <property name="buddy">
         <cstring>roomDescColorPushButton</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="roomDescColorLabel">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>MS Shell Dlg</family>
          <pointsize>8</pointsize>
          <italic>false</italic>
          <bold>true</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="text">
         <string>roomdesc</string>
        </property>
        <property name="margin">
         <number>3</number>
        </property>
        <property name="indent">
         <number>3</number>
        </property>
        <property name="buddy">
         <cstring>roomDescColorPushButton</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="roomDescColorPushButton">
        <property name="text">
         <string>Select</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Commands</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Character prefix:</string>
        </property>
        <property name="buddy">
         <cstring>charPrefixLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="charPrefixLineEdit">
        <property name="text">
         <string>_</string>
        </property>
        <property name="maxLength">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Emoji Shortcodes</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="encodeEmoji">
        <property name="text">
         <string>Encode to shortcodes</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="decodeEmoji">
        <property name="text">
         <string>Decode from shortcodes</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>341</width>
       <height>31</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>roomNameColorPushButton</tabstop>
  <tabstop>roomDescColorPushButton</tabstop>
  <tabstop>charPrefixLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
